src/Glpi/Http/RequestPoliciesTrait.php

```diff
@@ -74,11 +74,12 @@ protected function shouldCheckDbStatus(Request $request): bool
         }
 
         if (
-            \str_starts_with($path, '/install/')
-            || ($_SESSION['is_installing'] ?? false)
-            || ($_SESSION['is_updating'] ?? false)
+            // install/update endpoint
+            \str_starts_with(\strtolower($path), '/install/')
+            // `\Glpi\Controller\ProgressController::check()` route used during install/update process
+            || \str_starts_with(\strtolower($path), '/progress/check/')
```

Comment:
Isnt the progress check meant to be generic ? It won't be used only for install.